project.clj

```diff
@@ -100,6 +100,7 @@
 
   :profiles {:dev {:source-paths  ["dev"]
                    :dependencies  [[org.clojure/tools.namespace "0.2.5"]
+                                   [org.clojure/java.jmx "0.2.0"]
```

Comment:
The ideal thing would be to add it as a dep for the test jar, which maybe could be done via the test profile or something?  Failing that, adding it as a top-level dep seems better than where we are now.